app/src/main/java/org/coepi/android/ui/alertsdetails/AlertsDetailsViewModel.kt

```diff
@@ -16,10 +16,12 @@ class AlertsDetailsViewModel(
     val report: LiveData<List<AlertDetailsSymptomViewData>> = just(args.report.toViewData())
         .toLiveData()
 
-    // FIXME temporary title to identify the report. No design yet for this screen.
-    val title: LiveData<String> = just("Report: ${args.report.id}")
+    val title: LiveData<String> = just(args.report.contactTime.toMonthAndDay())
```

Comment:
You can add the line with the exposure time too. What we're leaving out is only the duration.